labkey-client-api/src/org/labkey/remoteapi/domain/PropertyDescriptor.java

```diff
@@ -67,6 +73,20 @@ public PropertyDescriptor(JSONObject json)
         _propertyURI = (String)json.get("propertyURI");
         _rangeURI = (String)json.get("rangeURI");
 
+        JSONArray cfs = (JSONArray) json.get("conditionalFormats");
+        cfs.forEach(cf -> {
+            JSONObject cfObj = (JSONObject) cf;
+            _conditionalFormats.add(
+                    new ConditionalFormat(
+                    (String) cfObj.get("filter"),
+                    (String) cfObj.get("textColor"),
+                    (String) cfObj.get("backgroundColor"),
+                    (Boolean) cfObj.get("bold"),
+                    (Boolean) cfObj.get("italic"),
+                    (Boolean) cfObj.get("strikethrough"))
+            );
```

Comment:
consider moving this code into the ConditionalFormat class, PropertyDescriptor shouldn't have to know how to deserialize a ConditionalFormat object, maybe a static factory method like:

static public ConditionalFormat fromJSON(JSONObject json)